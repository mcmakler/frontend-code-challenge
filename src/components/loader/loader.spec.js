import expect from 'expect';
import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import Loader from './loader';

describe('Loader specs', () => {
  it('should match snapshot', () => {

    const component = ReactTestRenderer.create(
      <Loader loaded={false}>
        <div>inside loader</div>
      </Loader>
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
