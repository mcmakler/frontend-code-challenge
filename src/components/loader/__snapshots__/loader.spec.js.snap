// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader specs should match snapshot 1`] = `
<div
  className="loader"
/>
`;
